mandatory/2-function-creation.js

```diff
@@ -6,7 +6,13 @@ Complete the function to check if the variable `num` satisfies the following req
 Tip: use logical operators
 */
 
-function validate(num) {}
+function validate(num) {
+  if (typeof num === "number" && num % 2 === 0 && num <= 100) {
```

Comment:
Fantastic, your logic here to validate is perfect. This is what we wanted to see and you delivered.